spec/models/bet_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Bet, type: :model do
+  let(:bob)   { User.create(username: 'Bob')   }
+  let(:peter) { User.create(username: 'Peter') }
+  let(:jack)  { User.create(username: 'Jack')  }
+
+  let(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  end
+
+  let(:option1) { BetOption.create(option_text: 'Atlanta wins') }
+  let(:option2) { BetOption.create(option_text: 'New England wins') }
+
+  let(:ub1) do
+    UserBet.create(user:       bob,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 100)
+  end
+
+  let(:ub2) do
+    UserBet.create(user:       peter,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 50)
+  end
+
+  let(:ub3) do
+    UserBet.create(user:       jack,
+                   bet:        bet1,
+                   bet_option: option2,
+                   amount_bet: 300)
+  end
+
+  let(:ub) do
+    ub1
+    ub2
+    ub3
+  end
+
+  context_string = 'Bet: Superbowl - Atlanta vs New England;\n' \
+    'Betters on Atlanta: Bob ($100), Peter ($50);\n' \
+    'Betters on New England: Jack ($300).'
+
+  context context_string do
```

Comment:
This context doesnt add a whole lot of value.  generally a `context` statement is used to change the test setup for specific edge cases:

```ruby
describe '#foo' do
  context 'on tuesdays this returns true' do
    let(:date) { :tuesday } 
    it { is_expected.to be truthy }
  end 
  context 'on mondays this returns false' do
    let(:date) { :monday }
    it { is_expected.to be truthy }
  end
end
```
This context is acting more as a description than changing the setup behaviour. If you feel your test needs to be more clear about what it doing, try and write your `let` statements so they are more descriptive about what they return